src/reference/00-Getting-Started/01-Setup/c.md

```diff
@@ -46,7 +46,8 @@ Run the following from the terminal to install `sbt` (You'll need superuser priv
 
     echo "deb https://repo.scala-sbt.org/scalasbt/debian all main" | sudo tee /etc/apt/sources.list.d/sbt.list
     echo "deb https://repo.scala-sbt.org/scalasbt/debian /" | sudo tee /etc/apt/sources.list.d/sbt_old.list
-    curl -sL "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823" | sudo apt-key add
+    curl -sL "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x2EE0EA64E40A89B84B2DF73499E82A75642AC823" | sudo -H gpg --no-default-keyring --keyring gnupg-ring:/etc/apt/trusted.gpg.d/scalasbt-release.gpg --import
+    sudo chmod 644 /etc/apt/trusted.gpg.d/scalasbt-release.gpg
```

Comment:
Does this require some version of GnuPG and Ubuntu?

Reply:
Good point @eed3si9n . I tried with `xenial`, `bionic` and `focal` Docker images and they require a combination of `apt-transport-https` (`xenial`), `curl` and `gnupg`. I will update it accordingly.

Reply:
@eed3si9n take a look